### PR TITLE
Update API.md to document submissions GET requests

### PR DESCRIPTION
**Pull request by kmcnellis**

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -635,6 +635,14 @@ Example Response:
 
 ## Submission API
 
+
+`GET /api/submissions` **Authenticated** - Gets all the submissions.  If `onlyInstructor=me` or `onlyInstructor=:userId` are specified, it will return only the submissions pertaining to the particular instructor specified by me or :userId. Likewise is `onlyStudent=me` or `onlyStudent=:userId` for students.  If `onlySection=:sectionId` is specified or `onlySectionEvent=:sectionEventId` is specified, only the submission related to the apprioiate section or section event are returned.
```
Comment (seveibar) [inline API.md:639]: *appropriate

File: API.md [hunk 2]
```diff
@@ -635,6 +635,14 @@ Example Response:
 
 ## Submission API
 
+
+`GET /api/submissions` **Authenticated** - Gets all the submissions.  If `onlyInstructor=me` or `onlyInstructor=:userId` are specified, it will return only the submissions pertaining to the particular instructor specified by me or :userId. Likewise is `onlyStudent=me` or `onlyStudent=:userId` for students.  If `onlySection=:sectionId` is specified or `onlySectionEvent=:sectionEventId` is specified, only the submission related to the apprioiate section or section event are returned.  
+
+Unless specified using `withStudents=false` and `withSectionEvent=false`, students and section events will be automatically populated.  If section events are populates, event info will also be, unless specified to not be returned wuth `withEventInfo=false`.
```
Comment (seveibar) [inline API.md:641]: *are populates

**Comment by seveibar** [comment]
excellent- much fewer typos ;)

**Comment by kmcnellis** [comment]
Did you fix the typos?

**Pull request merged**